index.js

```diff
@@ -15,6 +15,11 @@ var PluginError = gutil.PluginError;
 var PLUGIN_NAME = 'gulp-istanbul';
 var COVERAGE_VARIABLE = '$$cov_' + new Date().getTime() + '$$';
 
+function getNormalizedPath(filepath) {
```

Comment:
I guess I probably should have named it better... This is actually making sure the original platform file path separator is used. I think that `normalize-path` just makes everything use `/`, not sure if that would work or not. But I'll give it a try and see...

This isn't my project, so if it does work and you'd prefer to use an external dependency that's fine. But generally if what I'm needing from an external module is just a single line of code, it seems a bit of an excess to require external dependencies for that...